### PR TITLE
Both fibonacci works

### PR DESCRIPTION
**Pull request by malinaq3**

### CODE REVIEW

File: module1/homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,31 @@
 #pragma once
+#include<iostream>
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+
+
+int tab[100];
```
Comment (kontrybutor) [inline module1/homework/fibonacci/fibonacci.hpp:7]: I think you can change it to ```int tab [sequence]``` or even use a ```std::array``` :)

Comment (TalNovatzky) [inline module1/homework/fibonacci/fibonacci.hpp:7]: why do you need to use an array here?
you are allocating too much memory here - you can try to do it only with integers and loop.

Comment (malinaq3) [reply module1/homework/fibonacci/fibonacci.hpp]: I changed it to solution with only integers and loop. Thanks!

Comment (malinaq3) [reply module1/homework/fibonacci/fibonacci.hpp]: I don' know how to do it with `std::array`

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1430], "points": [2, 6] }

**Pull request closed without merging**